content/en/prio.html

```diff
@@ -8,16 +8,20 @@
     <div class="container">
         <h2>What is Prio?</h2>
         <p></p>
-        <p>Prio is a technology that technically enforces the privacy of application metrics.</p>
+        <p><a href=https://crypto.stanford.edu/prio/>Prio</a> is a privacy-respecting system for the collection of aggregate statistics such as application metrics.</p>
+
+        <p>In Prio, applications such as a web browser, mobile application or a website generate data, split it into two anonymized and encrypted shares shares, and upload each share to a different data share processor, such that neither processor can learn much about the original data. Each processor then aggregates its data shares into a partial sum, which can be combined into the final aggregation, permitting useful statistics over the whole body of data to be computed while revealing barely any information about individual participants, so long as either one of the data share processors does not disclose their data shares.</p>
```

Comment:
"while revealing barely any information"

Can you explain what's meant by "barely" here instead of just saying "no", an if we're going to keep the hedge word can we change it to "hardly" or "minimal"?

Reply:
I'm hedging here because of the wording from Stanford's website:
>As long as at least one server is honest, the Prio servers **learn nearly nothing** about the clients’ private data, except what they can infer from the aggregate statistics that the system computes.

I've changed the text to use "minimal information" in both places.